database/src/main/java/database/service/SqlDatabaseService.java

```diff
@@ -472,6 +609,44 @@ private String generateInsertSQL(BaseEntity entity, String tableName) {
         return "INSERT INTO " + tableName + " (" + fields + ") VALUES (" + placeholders + ")";
     }
 
+    private <T extends BaseEntity> String generateInsertMultipleLinesSQL(List<T> entities, String tableName) {
```

Comment:
Please check if we can move common code from this method and from `generateInsertSQL` into separate method.